src/main/flight/mixer.c

```diff
@@ -424,9 +424,11 @@ static void updateDynLpfCutoffs(timeUs_t currentTimeUs, float throttle)
 #endif
 
 static void applyMixerAdjustmentLinear(float *motorMix, const bool airmodeEnabled) {
-    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? motorMixRange : 1.0f;
+    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? 1.0 / motorMixRange : 1.0f;
     const float motorMixDelta = 0.5f * motorMixRange;
 
+    float minMotor = 1000.0;
+    float maxMotor = -1000.0;
```

Comment:
```suggestion
    float minMotor = FLT_MAX;
    float maxMotor = FLT_MIN;
```

You also need to `#include <float.h>` at the beginning of this file to use these macros.